content/bd/0004-relational-model-conversion.md

```diff
@@ -468,8 +460,8 @@ Neste caso, devemos criar uma _foreign key_ com a generalização imediatamente
 > associate(<u>name</u>, join_date)
 >
 > - name: FK(member)
-> - (IC-1) name must exist in 'bronze', 'silver' or 'gold'
-> - (IC-2) No associate can exist at the same time in any combination of 'bronze', 'silver' and 'gold'
+> - IC-1: _name_ must exist in 'bronze', 'silver' or 'gold'
+> - IC-2: _name_ cannot exist at the same time in 'bronze', 'silver' and 'gold'
```

Comment:
There is no example in the slides of a IC of 3 or more specializations, but I think the way this is written now conveys that it cannot be in all three simultaneously but it can be in two of them. Maybe swap the `and` with an `or`?

```suggestion
> - IC-2: _name_ cannot exist at the same time in 'bronze', 'silver' or 'gold'
```
@RafDevX thoughts?

Reply:
I don't know what the correct terminology is, but intuitively `or` sounds better than `and`.